cli/tests/integrations/test_package.py

```diff
@@ -667,11 +670,35 @@ def test_list_cli():
 
     stdout = file_json(
         'tests/data/package/json/test_list_helloworld_cli.json')
-    _list(stdout=stdout)
+    _list(args=['--json'], stdout=stdout)
 
     _uninstall_cli_helloworld()
 
 
+def test_list_cli_only():
+    helloworld_path = 'tests/data/package/json/test_list_helloworld_cli.json'
+    helloworld_json = file_json_ast(helloworld_path)[0]
+
+    def assert_test_case(args, expected_packages):
```

Comment:
we should use `assert_command` here instead. We can put in the expected stdout as a param.